LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesExpandableSetting.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  CorrectionRangeOverridesExpandableSetting.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 7/13/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import HealthKit
+import LoopKit
+
+
+public struct CorrectionRangeOverridesExpandableSetting<ExpandedContent: View>: View {
```

Comment:
This was copied pretty much as-is from `CorrectionRangeOverridesEditor`, so I may reuse it in `TherapySettingsView`